server/server.go

```diff
@@ -0,0 +1,26 @@
+package server
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+// CreateRouter creates and configures a server and returns a pointer a router with defined handles
+func CreateRouter() *gin.Engine {
+	router := gin.Default()
+	defineRoutes(router)
+	return router
+}
+
+// StartServer starts given server
+func StartServer(router *gin.Engine) {
+	srv := &http.Server{
+		Addr:    ":8080",
+		Handler: router,
+	}
+	if err := srv.ListenAndServe(); err != nil && err != http.ErrServerClosed {
+		log.Fatalf("listen: %s\n", err)
+	}
```

Comment:
Please later add channel to gracefully shutdown the server

Reply:
Created a new commit resolve this 5e7b5792fb7463807268c3cbdd223c94d2be4ce7